connectd/usr/bin/connectd_library

```diff
@@ -1111,7 +1111,7 @@ getNewPassword() {
            stty echo
         fi
 
-        if [ $(validateInput "$password" "\$ \@ \! \- \_ \. [a-zA-Z0-9]") != 0 ]; then
+        if [ $(validateInput "$password" "\$ \@ \! \_ \. [a-zA-Z0-9]-") != 0 ]; then
```

Comment:
I really don't like restricting the characters that can be used in password

Reply:
This change was just made so that it would work "as-is" with OpenWRT.  There have been many discussions about handling password validation in the API etc. but it hasn't gone anywhere.